.github/workflows/release.yaml

```diff
@@ -17,26 +17,37 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Check head commit message
-        env:
-          head_commit_title: ${{ github.event.head_commit.message }}
+        id: check_head_commit_message
+        continue-on-error: true
         run: |
-          python ./.github/scripts/validate_release_commit.py "$head_commit_title"
+          head_commit_message=`git log --format=%s -1`
+          python ./.github/scripts/validate_release_commit.py $head_commit_message
+          if [ $? -eq 1 ]
+          then
+            echo "valid_release_message=False" >> $GITHUB_OUTPUT
+          else
+            echo "valid_release_message=True" >> $GITHUB_OUTPUT
+          fi
 
       - name: Install commitizen
+        if: ${{ steps.check_head_commit_message.outputs.valid_release_message == 'True' }}
         run: |
           pip3 install commitizen
 
       - name: Setup Github Actions git user
+        if: ${{ steps.check_head_commit_message.outputs.valid_release_message == 'True' }}
         run: |
           git config --global user.email "airflow-oss-bot@astronomer.io"
           git config --global user.name "airflow-oss-bot"
 
       - name: Tag and push tag for triggering release
+        if: ${{ steps.check_head_commit_message.outputs.valid_release_message == 'True' }}
         run: |
           git tag `cz version -p`
           git push origin `cz version -p`
 
       - name: Bump version for new development
```

Comment:
Should we remove this step as it does not look complete. We're not committing/creating a PR here.